universe/ValueRefs.cpp

```diff
@@ -1613,6 +1674,26 @@ int ComplexVariable<int>::Eval(const ScriptingContext& context) const
         }
         return 0;
     }
+    else if (variable_name == "Named") {
```

Comment:
seems unnecessary to use an indirection through a ComplexVariable to look up the named ValueRef...

Reply:
Removed "Named" complex value ref completely